### ISSUE TITLE
[a11y]: Small toggle triggers "The SVG element has no accessible name"

### ISSUE THREAD

**Issue by jadamski02** [carbon-design-system/carbon#17676]
### Package

@carbon/react

### Browser

Chrome

### Operating System

MacOS

### Package version

1.63.2

### React version

17.0.2

### Automated testing tool and ruleset

IBM Equal Access Accessibility Checker

### Description

The error is triggered when _Toggle_ component has property size with value "sm" and it is on.
I managed to reproduce it in the storybook.

### Reproduction/example

https://stackblitz.com/edit/github-ojuhir?file=src%2FApp.jsx

### Steps to reproduce

1. Open Toggle component description on storybook
2. Go to playground and choose the "sm" value of "size" prop
3. Open the result in fullscreen
4. Scan it with IBM Equal Access Accessibility Checker

https://github.com/user-attachments/assets/491003c8-3a73-4134-81ff-cd93f679527e

### Suggested Severity

None

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by tay1orjones** [comment]
Thanks for reporting this! I put up a PR just now that should fix it.
